product_docs/docs/eprs/7/05_smr_operation/01_prerequisites/03_enable_access_to_database.mdx

```diff
@@ -17,34 +17,34 @@ No special steps are required to enable access to a Postgres database server.
 
 An Oracle JDBC driver `jar` file, such as `ojdbc8.jar`, must be accessible to the Java virtual machine (JVM) on the host running the publication server and the subscription server. If the publication server and subscription server are running on separate hosts, the Oracle JDBC driver must be accessible to the JVM on each host. Use Oracle JDBC driver version ojdbc8 or later.
 
-**Step 1:** Download the Oracle JDBC driver, for example, `ojdbc8.jar`, from the Oracle download site to the host that runs the publication server.
-
-**Step 2:** Copy the file `ojdbc8.jar` to the directory `XDB_HOME/lib/jdbc.`
-
-```shell
-$ su root
-Password:
-$ cd /usr/edb/xdb/lib/jdbc
-$ cp /home/user/Downloads/ojdbc8.jar .
-$ ls -l
-total 4032
--rw-rw-r-- 1 root root  355655 Jan 25 02:38 edb-jdbc14.jar
--rw-rw-r-- 1 root root  716209 Jan 25 02:38 edb-jdbc17.jar
--rw-rw-r-- 1 root root  317816 Jan 25 02:38 jtds-1.3.1.jar
--rw-r--r-- 1 root root 2091137 Jan 28 16:45 ojdbc8.jar
--rw-rw-r-- 1 root root  642809 Jan 25 02:38 postgresql-9.4-1201.jdbc4.jar
-```
+1. Download the Oracle JDBC driver, for example, `ojdbc8.jar`, from the Oracle download site to the host that runs the publication server.
 
-!!! Note
-    You can also copy the `ojdbc8.jar` file to the `jre/lib/ext` subdirectory of the location where you installed your Java runtime environment.
+1. Copy the file `ojdbc8.jar` to the directory `XDB_HOME/lib/jdbc.`
 
-!!! Note
-    Make sure to set the ODBC driver permission to a minimum of `644`.
+   ```shell
+   $ su root
+   Password:
+   $ cd /usr/edb/xdb/lib/jdbc
+   $ cp /home/user/Downloads/ojdbc8.jar .
+   $ ls -l
+   total 4032
+   -rw-rw-r-- 1 root root  355655 Jan 25 02:38 edb-jdbc14.jar
+   -rw-rw-r-- 1 root root  716209 Jan 25 02:38 edb-jdbc17.jar
+   -rw-rw-r-- 1 root root  317816 Jan 25 02:38 jtds-1.3.1.jar
+   -rw-r--r-- 1 root root 2091137 Jan 28 16:45 ojdbc8.jar
+   -rw-rw-r-- 1 root root  642809 Jan 25 02:38 postgresql-9.4-1201.jdbc4.jar
+   ```
 
-!!! Note
-    Make sure to copy `xdb6.jar` along with `ojdbc8.jar` at `/usr/edb/xdb/lib/jdbc/` if one or more tables in Oracle Publication contains an `XMLType` column when using Oracle to EDB Postgres Advanced Server/PostgreSQL replication.
+   !!! Note
+       You can also copy the `ojdbc8.jar` file to the `jre/lib/ext` subdirectory of the location where you installed your Java runtime environment.
+
+   !!! Note
+       Make sure to set the ODBC driver permission to a minimum of `644`.
+
+   !!! Note
+       Make sure to copy `xdb6.jar` along with `ojdbc8.jar` at `/usr/edb/xdb/lib/jdbc/` if one or more tables in Oracle Publication contains an `XMLType` column when using Oracle to EDB Postgres Advanced Server/PostgreSQL replication.
```

Comment:
@anwar-sahib @ccestes Is the `xdb6.jar` related instruction valid for Replication Server 7?  With "6" in the file name, it has me wondering if this jar file is specific to Replication Server 6.2.  If it is not relevant for Replication Server 7, then this note should be updated accordingly.